packages/apps-config/src/ui/logos/index.ts

```diff
@@ -306,6 +308,7 @@ export const nodeLogos: Record<string, unknown> = [
   ['Statemine Collator', nodeStatemine],
   ['Statemint Collator', nodeStatemine],
   ['SubDAO Collator', nodeSubDAO],
+  ['OriginTrail Parachain', nodeOriginTrail],
```

Comment:
Move up. 

(I should really add a test for these, with the fact that it is arrays the linter does not catch it)

Reply:
Done!